docs/resources/Channel.md

```diff
@@ -253,7 +258,7 @@ Represents a message sent in a channel within Discord.
 | Field       | Type      | Description                             |
 |-------------|-----------|-----------------------------------------|
 | message_id? | snowflake | id of the originating message           |
-| channel_id  | snowflake | id of the originating message's channel |
+| channel_id?  | snowflake | id of the originating message's channel |
```

Comment:
```suggestion
| channel_id? | snowflake | id of the originating message's channel |
```

As a side question, why are all three fields of this optional?

Reply:
This, wouldn't at least `message_id` be required

Reply:
message_id is not always present, see #2053 for details